projects/views.py

```diff
@@ -401,6 +415,32 @@ def view_project(request, project_id):
                     "An unexpected error occurred while attempting "
                     "to change role for this user. Please try again",
                 )
+        elif "su_budget_user" in request.POST:
+            budget_user = User.objects.get(username=request.POST["user_ref"])
+            budget_user_project = Project.objects.get(id=project.id)
+            enforced_by = User.objects.get(username=request.user.username)
+            charge_budget = set_budget_for_user_in_project(
+                budget_user, budget_user_project, request.POST["su_budget_user"], enforced_by
+            )
+            messages.success(
+                request,
+                (
+                    f"SU budget for user {budget_user.username} "
+                    f"is currently set to {charge_budget.su_budget}"
+                )
+            )
+        elif "default_su_budget" in request.POST:
```

Comment:
I'm not sure this works, because if anyone joins the project late, they won't get a budget.